pkg/pool-weighted/contracts/smart/ManagedPool.sol

```diff
@@ -355,6 +362,35 @@ contract ManagedPool is BaseWeightedPool, ReentrancyGuard {
         emit SwapEnabledSet(swapEnabled);
     }
 
+    /**
+     * @dev Getter for the sum of all weights. In initially FixedPoint.ONE, it can be higher or lower
+     * as a result of adds and removes.
+     */
```

Comment:
I added this both for general purpose visibility (e.g., debugging/testing), and in case it might be valuable for the SOR or UI to quickly check if it's non-one in case they need to process those differently for some reason. Will add the comment.